File: Fullstack/2d place Serhii Chenakal/project1/models/Flight.php
<?php

namespace app\models;

use Yii;

/**
 * This is the model class for table "flights".
 *
 * @property integer $id
 * @property string $start
 * @property integer $duration
 * @property string $origin
 * @property string $destination
 * @property double $cost
 */
class Flight extends \yii\db\ActiveRecord
{
    /**
     * @inheritdoc
     */
    public static function tableName()
    {
        return 'flights';
    }

    /**
     * @inheritdoc
     */
    public function rules()
    {
        return [
            [['start'], 'safe'],
            [['duration'], 'integer'],
            [['cost'], 'number'],
            [['origin', 'destination'], 'string', 'max' => 255],
            [['start', 'duration', 'cost', 'origin'], 'required'],
        ];
    }

    /**
     * @inheritdoc
     */
    public function attributeLabels()
    {
        return [
            'id' => 'ID',
            'start' => 'Start',
            'duration' => 'Duration',
            'origin' => 'Origin',
            'destination' => 'Destination',
            'cost' => 'Cost',
        ];
    }

    /**
     * @inheritdoc
     * @return FlightQuery the active query used by this AR class.
     */
    public static function find()
    {
        return new FlightQuery(get_called_class());
    }
}

File: Fullstack/3d place Anatolii Antonenko/dev-final/src/main/java/app/model/Flight.java
package app.model;

import java.time.Duration;
import java.time.LocalTime;

public class Flight {
    private long id;
    private LocalTime start;
    private Duration duration;
    private CityNode origin;
    private CityNode destination;
    private long cost;

    @Override
    public String toString() {
        return "Flight{" +
                "id=" + id +
                ", start=" + start +
                ", duration=" + duration +
                ", origin=" + origin +
                ", destination=" + destination +
                ", cost=" + cost +
                '}';
    }

    public Flight(long id, LocalTime start, Duration duration, CityNode origin, CityNode destination, int cost) {
        this.id = id;
        this.start = start;
        this.duration = duration;
        this.origin = origin;
        this.destination = destination;
        this.cost = cost;
    }

    public long getId() {
        return id;
    }

    public Duration getDuration() {
        return duration;
    }

    public long getCost() {
        return cost;
    }

    public CityNode getOrigin() {
        return origin;
    }

    public CityNode getDestination() {
        return destination;
    }

    public void setCost(int cost) {
        this.cost = cost;
    }
}
